spark/src/main/java/org/opensearch/sql/spark/execution/statement/StatementId.java

```diff
@@ -13,7 +13,7 @@ public class StatementId {
   private final String id;
 
   public static StatementId newStatementId() {
-    return new StatementId(RandomStringUtils.random(10, true, true));
+    return new StatementId(RandomStringUtils.random(16, true, true));
```

Comment:
change length from 10 to 16, making it similar to EMR-S job id.

Reply:
Nit::  randomAlphaNumeric

Reply:
done.